src/inc/msquic.h

```diff
@@ -1631,6 +1647,8 @@ typedef struct QUIC_API_TABLE {
     QUIC_CONNECTION_COMP_RESUMPTION_FN  ConnectionResumptionTicketValidationComplete; // Available from v2.2
     QUIC_CONNECTION_COMP_CERT_FN        ConnectionCertificateValidationComplete;      // Available from v2.2
 
+    QUIC_STREAM_PROVIDE_RECEIVE_BUFFERS_FN
+                                        StreamProvideReceiveBuffers; // Available from v2.2
```

Comment:
Next API version will be 2.5
```suggestion
                                        StreamProvideReceiveBuffers; // Available from v2.5
```